README.md

```diff
@@ -44,6 +44,24 @@ With the IBM Cloud Operator, you can provision and bind [IBM public cloud servic
 [Back to top](#ibm-cloud-operator)
 <!-- SHOW operator hub -->
 
+## Upgrading the operator
+To upgrade, you can reinstall the operator through the OperatorHub or the `curl` [installation command](README.md#setting-up-the-operator).
```

Comment:
This one may need to be careful to distinguish the install processes a bit. Since Operator Hub installs in a very different way than the install script does.